app/managers.py

```diff
@@ -0,0 +1,41 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        actor_cursor = self._connection.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"
+        )
+
+        return [Actor(*row) for row in actor_cursor]
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name,last_name) VALUES (?,?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def update(self, actor_id: int, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            f"SET first_name=?, last_name=? "
```

Comment:
also sometimes you write `first_name=?`, sometimes `first_name = ?`. Make it `first_name = ?` everywhere please

Reply:
your code should be one styled

Reply:
Fixed all .